lib/lambda_ethereum_consensus/state_transition/operations.ex

```diff
@@ -174,7 +174,56 @@ defmodule LambdaEthereumConsensus.StateTransition.Operations do
     Enum.reverse(withdrawals)
   end
 
-  @doc """
+  @spec process_attester_slashing(BeaconState.t(), SszTypes.AttesterSlashing.t()) ::
+          {:ok, BeaconState.t()} | {:ok, nil}
+  def process_attester_slashing(state, attester_slashing) do
+    attestation_1 = attester_slashing.attestation_1
+    attestation_2 = attester_slashing.attestation_2
+
+    cond do
+      Predicates.is_slashable_attestation_data(attestation_1.data, attestation_2.data) == false ->
+        {:ok, nil}
```

Comment:
This should return an error.

Reply:
Spec tests must be altered for operations for me to return an error since it won't fit with the spec test. I was handling with the errors but expected return value was {:ok, nil} when there is an error. Don't know how can I fix it this way

Reply:
Inherited implementation from the previous handlings

Reply:
Inherited implementation from the previous handlings